### PR TITLE
Add autoyast profiles for default modules all patterns in sle15sp5 migration

### PR DESCRIPTION
**Pull request by JRivrain**
See https://progress.opensuse.org/issues/135683:
Create support image via autoyast with default modules and all patterns on SLE 15 SP5

VRs https://openqa.suse.de/tests/overview?build=JRivrain%2Fos-autoinst-distri-opensuse%2317915&version=15-SP5&distri=sle
VRs for migration jobs https://openqa.suse.de/tests/overview?build=JRivrain%2Fos-autoinst-distri-opensuse%2317915&version=15-SP6&distri=sle
MR https://gitlab.suse.de/qa-maintenance/qam-openqa-yml/-/merge_requests/629/diffs

### CODE REVIEW

File: data/yam/autoyast/support_images/sles15sp5_install_gnome_all_patterns_s390.xml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        <service>firewalld</service>
+        <service>wicked</service>
+	      <service>sshd</service>
+        <service>systemd-remount-fs</service>
+      </enable>
+    </services>
+  </services-manager>
+  <software t="map">
+    <patterns t="list">
+          <pattern>apparmor</pattern>
```
Comment (lemon-suse) [inline data/yam/autoyast/support_images/sles15sp5_install_gnome_all_patterns_s390.xml:78]: Please update indent to make it align with other <pattern>.

File: data/yam/autoyast/support_images/sles15sp5_install_gnome_all_patterns_aarch64.xml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    mv /var/run/zypp.sav /var/run/zypp.pid
+
+exit 0
+
+]]></source>
+      </script>
+    </post-scripts>
+  </scripts>
+  <services-manager t="map">
+       <default_target>graphical</default_target>
```
Comment (lemon-suse) [inline data/yam/autoyast/support_images/sles15sp5_install_gnome_all_patterns_aarch64.xml:39]: indent align issue.

File: data/yam/autoyast/support_images/sles15sp5_install_gnome_all_patterns_aarch64.xml [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    <services t="map">
+      <disable t="list"/>
+      <enable t="list">
+        <service>firewalld</service>
+        <service>wicked</service>
+	      <service>sshd</service>
```
Comment (lemon-suse) [inline data/yam/autoyast/support_images/sles15sp5_install_gnome_all_patterns_aarch64.xml:45]: indent align issue.

**Comment by lemon-suse** [comment]
In general, LGTM, just need update these profiles for indent align issue. And please provide VRs of migration test.

**Comment by lemon-suse** [comment]
About the PR title 'Create 2 new autoyast profiles for sle15 migration', how do you think to let it be 'Add autoyast profiles of default module all patterns for sle15sp5 migration' ?

File: data/yam/autoyast/support_images/sles15sp5_install_gnome_all_patterns_aarch64.xml (hunk 3 above)
Comment (JRivrain) [reply data/yam/autoyast/support_images/sles15sp5_install_gnome_all_patterns_aarch64.xml:45]: it's very strange, this appears aligned in vscode...

Comment (JRivrain) [reply data/yam/autoyast/support_images/sles15sp5_install_gnome_all_patterns_aarch64.xml]: corrected in vi...

**Comment by rfan1** [comment]
@JRivrain I can use your ay files as template for sle15sp4 support image :)

**Review by rfan1 — APPROVED** [review]
LGTM

**Review by lemon-suse — APPROVED** [review]

**Comment by lemon-suse** [comment]
So just need migration VRs for merge.

**Comment by JRivrain** [comment]
> So just need migration VRs for merge.

Can you please indicate which jobs are meant to use these images ?

**Comment by lemon-suse** [comment]
> > So just need migration VRs for merge.
> 
> Can you please indicate which jobs are meant to use these images ?

The ticket to add test suites blocked for the new image not ready, so you can't find all of them from migration job group. Two ways to do it, 1. select one existing 15SP3 test in migration daily job group and clone it with correct setting such as default module and all patterns, set HDD_1 as your new qcows.  2.create such test suites in your own development group and trigger, for this you can refer LEO's VR https://openqa.suse.de/tests/overview?build=issues-132848 for Flavor: Migration-from-SLE15-SPx.

**Comment by rfan1** [comment]
@JRivrain one comment about the ay file name for s390x platform:

data/yam/autoyast/support_images/sles15sp5_install_gnome_all_patterns_**s390**.xml -> data/yam/autoyast/support_images/sles15sp5_install_gnome_all_patterns_**s390x**.xml

**Comment by JRivrain** [comment]
> @JRivrain I can use your ay files as template for sle15sp4 support image :)


Great, I saw from you VRs that those also work for sp4. I thought to name those profiles like sles15spx but then realized these will not work for sp3 or earier since they include python3. then I though to name it sp4+ but then, we don't know, we may use these also for sp6 in the future. So well, it would require more discussion and we want this merged for now. Feel free to use those and rename them later to something that makes more sense, if you want. Or just use them as-is for sp4, that's not a very big deal after all :)

**Comment by JRivrain** [comment]
> > > So just need migration VRs for merge.
> > 
> > 
> > Can you please indicate which jobs are meant to use these images ?
> 
> The ticket to add test suites blocked for the new image not ready, so you can't find all of them from migration job group. Two ways to do it, 1. select one existing 15SP3 test in migration daily job group and clone it with correct setting such as default module and all patterns, set HDD_1 as your new qcows. 2.create such test suites in your own development group and trigger, for this you can refer LEO's VR https://openqa.suse.de/tests/overview?build=issues-132848 for Flavor: Migration-from-SLE15-SPx.

Thank you, VRs added.

**Comment by rfan1** [comment]
> > @JRivrain I can use your ay files as template for sle15sp4 support image :)
> 
> Great, I saw from you VRs that those also work for sp4. I thought to name those profiles like sles15spx but then realized these will not work for sp3 or earier since they include python3. then I though to name it sp4+ but then, we don't know, we may use these also for sp6 in the future. So well, it would require more discussion and we want this merged for now. Feel free to use those and rename them later to something that makes more sense, if you want. Or just use them as-is for sp4, that's not a very big deal after all :)

Yes, I can copy a new name or create a soft-link then :)

**Pull request merged**